tests/instruments/hp/test_hp8753e_with_device.py

```diff
@@ -40,17 +40,23 @@ def hp8753e(connected_device_address):
     try:
         # This allows a PrologixAdapter to be used for running the tests as well
         #  `pytest --device-address PrologixAdapter,ASRL4::INSTR,16 -k "test_hp8753e_with_device"`
+
         if "PrologixAdapter" not in connected_device_address:
             vna = HP8753E(connected_device_address)
         else:
             _, prologix_address, gpib_address, *other_address_info = connected_device_address.split(
                 ","
             )
-            prologix = PrologixAdapter(resource_name=prologix_address, visa_library="@py", auto=1)
+
+            temp_prologix = PrologixAdapter(
+                resource_name=prologix_address, visa_library="@py", auto=1
+            )
+            prologix = temp_prologix.gpib(gpib_address)
```

Comment:
Fixed. Thank you for being patient with me fixing the issues identified in my code. Hopefully the next instrument (Keysight E5071C) will go a bit smoother.

I think the functionality you exposed with this PR will help me to utilize inheritance for the multitude of shared functionality of the 5061 and 5071 VNAs since they can be configured with 2 or 4 ports and a plethora of different frequency ranges.

Reply:
Thanks for considering more instruments and thanks for your patience and working along with this review process.